README.md

```diff
@@ -3,7 +3,7 @@
 1. Fork or download the [template](https://github.com/curtisbarnard/BANKI/blob/main/BANKI-template.md)
 2. Start filling in your answers.
    - If you are having trouble with an answer take a look at the [completed sheet](https://github.com/curtisbarnard/BANKI/blob/main/BANKI-answers.md) for resources/inspiration.
-3. For each answer immediately add it to your ANKI deck and start reviewing.
+3. For each answer immediately add it to your BANKI deck and start reviewing.
```

Comment:
I think this should actually be Anki, but perhaps with a link to here: https://apps.ankiweb.net/ Would you like to update it?